crates/router/src/connector/stripe/transformers.rs

```diff
@@ -280,6 +280,68 @@ pub struct StripeBankRedirectData {
     pub bank_specific_data: Option<BankSpecificData>,
 }
 
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct AchBankTransferData {
+    #[serde(rename = "owner[email]")]
+    pub email: Email,
+}
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct BacsBankTransferData {
+    #[serde(rename = "payment_method_data[type]")]
+    pub payment_method_data_type: StripePaymentMethodType,
+    #[serde(rename = "payment_method_data[billing_details][email]")]
+    pub email: Email,
+    #[serde(rename = "payment_method_data[billing_details][name]")]
+    pub name: String,
```

Comment:
These fields are already there in 
https://github.com/juspay/hyperswitch/blob/a6cc8e08c560bec2e59c7c7290a6382d1611e8df/crates/router/src/connector/stripe/transformers.rs#L1723 can reuse the same